### PR TITLE
Lcm of two numbers

### PR DESCRIPTION
**Pull request by tk315**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Article on LCM of two number addded

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#249

### CODE REVIEW

**Comment by tk315** [comment]
Please review sir @HarshCasper @iamrajiv @ankitaggarwal23 @atarax665 @gat786 @nidh-ish

File: docs/mathematics/lcm-of-two-numbers.mdx [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+
+## Introduction
+
+Lowest Common Multiple(LCM) is the lowest number which is divisible by both the numbers. It is also called as Least Common Divisor(LCD). LCM of two numbers cannot be less than the greater of them. If smaller number is the factor of greater number then the greater number will be the LCM.
+
+## Explanation 
+
+LCM of two numbers `a` and `b` is denoted by `lcm(a,b)`. LCM of any number with `1` will be the number itself. LCM of two coprime numbers is the product of numbers. LCM can be found by Listing Multiples, Prime Factorization, Ladder Method, and Divison Method.
+
+LCM has following properties:
```
Comment (nidh-ish) [inline docs/mathematics/lcm-of-two-numbers.mdx:14]: *has the following

File: docs/mathematics/lcm-of-two-numbers.mdx [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+Iteration 7:
+maxx = 12
+maxx is divisible by both the numbers, so this is our LCM
+```
+
+## Algorithm
+
+- Declare a variable `maxx` and initialize it to maximum of two numbers.
+
+- Increase `maxx` by `1` untill it become divisible by both the numbers.
```
Comment (nidh-ish) [inline docs/mathematics/lcm-of-two-numbers.mdx:66]: *until it becomes

Remove extra lines in between.

File: docs/mathematics/lcm-of-two-numbers.mdx [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+#### Output :
+
+```
+LCM of two Numbers is: 15
+```
+
+## Complexity Analysis
+
+- Time Complexity: `O(n)`
+
```
Comment (nidh-ish) [inline docs/mathematics/lcm-of-two-numbers.mdx:170]: Remove this extra line.

**Review by nidh-ish — CHANGES_REQUESTED** [review]

**Comment by nidh-ish** [comment]
Also, @tk315 resolve the merge conflicts.

File: docs/mathematics/lcm-of-two-numbers.mdx (hunk 1 above)
Comment (tk315) [reply docs/mathematics/lcm-of-two-numbers.mdx]: done

File: docs/mathematics/lcm-of-two-numbers.mdx (hunk 2 above)
Comment (tk315) [reply docs/mathematics/lcm-of-two-numbers.mdx]: done

File: docs/mathematics/lcm-of-two-numbers.mdx (hunk 3 above)
Comment (tk315) [reply docs/mathematics/lcm-of-two-numbers.mdx]: done

**Comment by tk315** [comment]
@nidh-ish all requested changes done

**Review by HarshCasper — APPROVED** [review]

**Pull request merged**